### PR TITLE
Installer architecture warning for 32-bit machines

### LINKED ISSUE #4: Release InVEST as a 64-bit binary
When we started developing and shipping binaries for InVEST, 32-bit binaries was an obvious choice as so many computers were still running Windows XP or were running on very old hardware.  In this modern age, I can't remember the last time I saw someone working with a 32-bit windows computers, and macs are all 64-bit anyways.  The only computer system I can think of that runs a 32-bit OS is the Raspberry Pi, but that'll be on linux and would therefore not be using our prebuilt binaries.

- [x] Check the InVEST usage stats ... do we have any records of people running InvEST on a 32-bit system in recent history?
- [x] If it's reasonable to only ship 64-bit, we should still run our tests against 32-bit and 64-bit builds of python on Windows if possible.

### PR DESCRIPTION
**Pull request by dcdenu4**
This PR adds a warning message to the InVEST installer on the landing page. The message warns 32-bit computer users that the 64-bit software being installed probably won't work. 

installer-warning-screenshot-small

Fixes #4

### CODE REVIEW

**Review by phargogh — COMMENTED** [review]
Looks good to me @dcdenu4 !  I'm unable to test the presence of the alert, buy what's here makes sense to me.  I'm leaving a comment here because I thought this might be a good opportunity to update a few known references to the -x86 path suffix in this repo.  What do you think?

File: installer/windows/invest_installer.nsi [hunk 1]
```diff
@@ -274,7 +289,7 @@ Function Un.onInit
 FunctionEnd
 
 ; Copied into the invest folder later in the NSIS script
-!define INVEST_BINARIES "$INSTDIR\invest-3-x86"
+!define INVEST_BINARIES "$INSTDIR\invest-3-x64"
```
[the remark this answers is not in the document]
Comment (dcdenu4) [reply installer/windows/invest_installer.nsi:292]: `getting_started` page updated here: https://github.com/natcap/invest.users-guide/commit/f414425390a790867e076c38a5400fe13c59d812

[the remark this answers is not in the document]
Comment (dcdenu4) [reply installer/windows/invest_installer.nsi:292]: Update to `invest-autotest.bat` and `cli.rst`: https://github.com/natcap/invest/pull/366/commits/b5abc5c3db5ea25a246d22d7edb07207fee56a86

**Comment by dcdenu4** [comment]
Alright @phargogh , this should be ready for another review.

**Review by phargogh — APPROVED** [review]

**Pull request merged**